packages/grid/_modules_/grid/models/api/gridVirtualizationApi.ts

```diff
@@ -17,7 +18,7 @@ export interface GridVirtualizationApi {
    * @param params
    * @returns boolean Return if the index was outside of the viewport and the grid has to scroll to reach the target.
    */
-  scrollToIndexes: (params: GridCellIndexCoordinates) => boolean;
+  scrollToIndexes: (params: Optional<GridCellIndexCoordinates, 'rowIndex'>) => boolean;
```

Comment:
A bit difficult to get why rowIndex becomes optional. Maybe we can add GridColumnHeaderIndexCoordinates? 🤔

Reply:
It because option so it can `scrollToIndexes` can handle horizontal scrolling only.

Reply:
yes in the context of the column header. So the type should be `GridCellIndexCoordinates | GridColumnHeaderIndexCoordinates`

Reply:
Yes, it's either that or what you said. The problem with the `GridCellIndexCoordinates | GridColumnHeaderIndexCoordinates` type is that the type needs to be explicit everywhere `rowIndex` is used, so `param.rowIndex as GridCellIndexCoordinates`.